source/jormungandr/jormungandr/authentication.py

```diff
@@ -199,7 +199,7 @@ def uncached_get_user(token):
 
         # if user doesn't exist for a token, get default token with user_type = no_access
```

Comment:
Maybe this comment is not appropriate anymore.

Reply:
Done